ui/src/app/utils/research-purpose.tsx

```diff
@@ -1,9 +1,13 @@
 import * as React from 'react';
 
-import {ResearchPurposeItems, SpecificPopulationItems} from 'app/pages/workspace/workspace-edit-text';
-import {PrimaryPurposeItems, ResearchPurposeItem} from 'app/pages/workspace/workspace-edit-text';
+import {
+  PrimaryPurposeItems,
+  ResearchPurposeItem,
+  ResearchPurposeItems,
+  SpecificPopulationItems
```

Comment:
Existing issue - flagging this for a follow-up: the only import here which is correctly capitalized is `ResearchPurposeItem`, since it's a type. The rest of these are just constants which should not be capitalized. This is adding another layer of confusion to the code.